index.html

```diff
@@ -1,72 +1,119 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>Vaccine Impact Modelling Consortium - Montagu</title>
-        <link rel="stylesheet" href="./resources/style.css" />
-    </head>
-    <body>
+<head>
+    <title>Vaccine Impact Modelling Consortium - Montagu</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
```

Comment:
Since I'm not using webpack I've ended up with a motley collection of CDN links and local copies of third party js. Use CDN where possible, or keep everything localy?

Reply:
I'd say have everything locally. Knowing how frequently js packages change, I'd say optimal to control which versions we're including!

Also rather than checking in all the resources, can we get them from npm and copy them into place when building the image?